cli/internal/taskhash/taskhash_test.go

```diff
@@ -29,16 +30,16 @@ func Test_manuallyHashPackage(t *testing.T) {
 		hash     string
 	}
 	files := map[string]fileHash{
-		filepath.FromSlash("top-level-file"):              {"top-level-file-contents", ""},
-		filepath.FromSlash("other-dir/other-dir-file"):    {"other-dir-file-contents", ""},
-		filepath.FromSlash("ignoreme"):                    {"anything", ""},
-		filepath.FromSlash("libA/some-file"):              {"some-file-contents", "7e59c6a6ea9098c6d3beb00e753e2c54ea502311"},
-		filepath.FromSlash("libA/some-dir/other-file"):    {"some-file-contents", "7e59c6a6ea9098c6d3beb00e753e2c54ea502311"},
-		filepath.FromSlash("libA/some-dir/another-one"):   {"some-file-contents", "7e59c6a6ea9098c6d3beb00e753e2c54ea502311"},
-		filepath.FromSlash("libA/ignoreme"):               {"anything", ""},
-		filepath.FromSlash("libA/ignorethisdir/anything"): {"anything", ""},
-		filepath.FromSlash("libA/pkgignoreme"):            {"anything", ""},
-		filepath.FromSlash("libA/pkgignorethisdir/file"):  {"anything", ""},
+		"top-level-file":              {"top-level-file-contents", ""},
```

Comment:
These paths are passed to `os.Create` below. This works because windows accepts `/`-delimited paths. However, I think we should convert at the point of interfacing w/ the filesystem so that we're clear. 

Perhaps typing this map as `map[RelativeUnixPath]fileHash` would highlight this?

Reply:
Ah, good catch, I caught just one of the ways this was working; we need to get the utility functions from `fs.AbsolutePath` merged into these so this sort of mistake doesn't happen.

(We were using it for creation of the files and wanted System paths, but also using it to check the outputs where we want `AnchoredUnixPaths`.)